Readme.md

```diff
@@ -75,6 +75,47 @@ install process be as simple as the `brew` process on macOS. See
 [#433](https://github.com/microsoft/scalar/issues/433) for progress on
 this feature._
 
+Installing on Linux
+-------------------
+
+Currently, we package a custom version of Git and Scalar as `.deb` packages
+that can be installed after downloading from the GitHub releases pages. The
+latest releases can be downloaded and installed as follows:
+
+```sh
+# Install microsoft/git
```

Comment:
```suggestion
# Install git-vfs
```

Might be better to go with the actual name of the package here?

Reply:
I was using the GitHub fork names. What is "the package"? In `brew`, this is `microsoft-git`, right?